### PR TITLE
Form error messages to use localisation

### PR DESCRIPTION
**Pull request by knasher**

### CODE REVIEW

File: src/TransformCore/Bundle/CsrFastStreamBundle/Resources/config/validation.yml [hunk 1]
```diff
@@ -8,47 +8,69 @@ TransformCore\Bundle\CsrFastStreamBundle\Entity\Applicant:
...
+            minMessage: error.plainpassword.length.min
+        - NotBlank:
             groups: [CsrRegistration, CsrPassword]
+            message: error.plainpassword.notblank
         - Regex:
-            pattern: '/\d/'
-            message: Must contain at least 1 number
             groups: [CsrRegistration, CsrPassword]
-        - Regex:
+            message: error.plainpassword.regex.letter
```
Comment (eddiejaoude) [inline src/TransformCore/Bundle/CsrFastStreamBundle/Resources/config/validation.yml]: Should `plain password` be camelcase?

File: src/TransformCore/Bundle/CsrFastStreamBundle/Resources/config/validation.yml [hunk 2]
```diff
@@ -8,47 +8,69 @@ TransformCore\Bundle\CsrFastStreamBundle\Entity\Applicant:
...
+            pattern: '/[!@#$%^*_-]+/'
       dateOfBirth:
-        - NotBlank: { groups: [CsrProfile] }
-        - Date: { groups: [CsrProfile] }
+        - Date:
+            groups: [CsrProfile]
+            message: error.dateofbirth.date
+        - NotBlank:
+            groups: [CsrProfile]
+            message: error.dateofbirth.notblank
```
Comment (eddiejaoude) [inline src/TransformCore/Bundle/CsrFastStreamBundle/Resources/config/validation.yml]: camelcase?

**Comment by knasher** [comment]
Could someone review?

**Comment by eddiejaoude** [comment]
Looks good.

Which branch are the actual Translations files in?

**Comment by eddiejaoude** [comment]
Found it https://github.com/TransformCore/CSR-Fast-Stream-frontend/pull/52/files

**Pull request merged**